### PR TITLE
added infrastructure info

### PR DESCRIPTION
**Pull request by kyokukou**

### CODE REVIEW

**Comment by norbusan** [comment]
I changed the title a bit for typos

File: README.md [hunk 1]
```diff
@@ -43,4 +43,15 @@ note that the data is base 64 encoded, and that return values from cloud functio
         "subscription": "projects/MY-PROJECT/subscriptions/MY-SUB"
       }'
     
- ```
\ No newline at end of file
+ ```
+
+ # deploying
+  in GCP your function will need a trigger to trigger the build of the function on deploy, a service account to run the build, a service account to run the function, and well as a pubsub subscriber (or some other event) to drive it. Each function needs its own subscriber
```
Comment (norbusan) [inline README.md]: Should be `as well as a pubsub` instead of `and well ..`

**Review by norbusan — COMMENTED** [review]
Honestly, reading this it does not really help me with deployments :-(

**Comment by kyokukou** [comment]
> Honestly, reading this it does not really help me with deployments :-(

Is there something that would be more helpful? There is a hello-world version for each of the pieces needed in the development project that people can use as a model. But I'm not sure I'm confident enough my entire cloud function process is the best/only way to do it so I'm trying to avoid writing a how-to guide on it at this point.

**Review by norbusan — APPROVED** [review]
LGTM

**Pull request merged**